src/ControlSystem/Systems/Translation.hpp

```diff
@@ -87,16 +88,22 @@ struct Translation : tt::ConformsTo<protocols::ControlSystem> {
 
   struct process_measurement {
```

Comment:
Maybe not right now, but how hard would factoring out all these "just forward the centers" `process_measurement` structs be?  They all look pretty similar.

Reply:
I think we could by templating on the system (Rotation, Expansion, Translation); probably a conversation to have with @knelli2 as well.

Reply:
Yeah we can probably add it to the `Measurements/` directory and template it on the control system like Francois said. At least for the simple (rot, exp, trans) control systems that is

Reply:
Right.  Anything that cares about more than the centers will have to implement its own thing.